Docs/deep_linking.md

```diff
@@ -0,0 +1,169 @@
+
+## Common parameters:
+### objectGroup
+
+This paramater can have values:
+  - BhaRun
+  - ChangeLog
+  - FluidsReport
+  - FormationMarker
+  - Message
+  - MudLog
+  - Log
+  - Rig
+  - Risk
+  - Trajectory
+  - Tubular
+  - WbGeometry
+
+### logType
+
+This paramater can have values:
+  - time
+  - depth
+
+### filterType
+
+This paramater can have values:
+  - Wellbore
+  - Log
+  - Rig
+  - Service%20Company
+
+## Navigation paths:
+
+### Wells View
+/servers/:serverUrl/wells
+
+### Wellbores View
+/servers/:serverUrl/wells/:wellUid/wellbores
+
+### Object Groups
+/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups
+
+### Objects View
+/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/objects
+
+### Object View
+/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/objects/:objectUid
+
+### Log Types View
+/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/logtypes
+
+### Log Objects View
+/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/logtypes/:logType/objects
+
+### Log Object View
+/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/logtypes/:logType/objects/:objectUid
+
+### Filtering Rows Of Views
+
+Most pages can receive a filter to filter the rows of the table in that page. Those filters are added at the end of the url.
+For example showing only rows where parameter "name" has value "test":
+
+?filter={"name":"test"}
+
+Url encoded version which needs to be used:
+
+%3Ffilter%3D%7B%22name%22%3A%22test%22%7D
+
+
+### Log Curve Values View
+/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/logtypes/:logType/objects/:objectUid/curvevalues
+
+#### Additional params for searching
+
+If there is a need to search in result, following parameters need to be filled up:
+
+  - startIndex
+  - endIndex
+  - mnemonics
+
+StartIndex and endIndex could be numbers for depth based logs or datetime in ISO format  - "YYYY-MM-DDTHH:mm:ss.sssZ" for time based logs
+
+Samples:
+
+##### Time sample:
+?startIndex=2024-09-06T00:00:00Z&endIndex=2024-09-06T00:01:00Z&mnemonics=["mnem1","mnem2","mnem3","mnem4"]
+
+url encoded version which should be used:
+%3FstartIndex%3D2024-09-06T00%3A00%3A00Z%26endIndex%3D2024-09-06T00%3A01%3A00Z%26mnemonics%3D%5B%22mnem1%22%2C%22mnem2%22%2C%22mnem3%22%2C%22mnem4%22%5D
+
+In the sample above we combine mnemonics "mmem1", "mmem2", "mmem3", "mmen4" for the selected time interval
+
+##### Depth sample:
+
+?startIndex=1&endIndex=4&mnemonics=["mnem1","mnem2","mnem3","mnem4"]
+
+url encoded version which should be used:
+%3FstartIndex%3D1%26endIndex%3D4%26mnemonics%3D%5B%22mnem1%22%2C%22mnem2%22%2C%22mnem3%22%2C%22mnem4%22%5D
+
+### Multi Logs Curve Info List View
+/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/logtypes/:logType/multilogs"
```

Comment:
Add the search params expected here also (like `?logs=<log uids>`)

Reply:
done